Samples/NetworkMediaStreamer/CameraRTPStreamerApp/CameraRTPStreamerApp.cpp

```diff
@@ -0,0 +1,419 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+
+#include <iostream>
+#include <iomanip>
+#include <fstream>
+#include <Mferror.h>
+#include <mfidl.h>
+#include <mfreadwrite.h>
+#include <mfapi.h>
+#include <windows.media.h>
+#include <windows.media.core.interop.h>
+#include <windows.foundation.h>
+#include <windows.Storage.streams.h>
+#include <winrt\base.h>
+#include <winrt\Windows.Media.Capture.h>
+#include <winrt\Windows.Media.Capture.Frames.h>
+#include <winrt\Windows.Graphics.Imaging.h>
+#include <winrt\Windows.Foundation.h>
+#include <winrt\Windows.Devices.Enumeration.h>
+#include <winrt\Windows.Media.Devices.h>
+#include <winrt\Windows.Media.MediaProperties.h>
+#include <winrt\Windows.Networking.Connectivity.h>
+#include <winrt\Windows.storage.streams.h>
+
+#include "..\Common\inc\NetworkMediaStreamer.h"
+#include "..\Common\inc\RTPMediaStreamer.h"
+#include "..\Common\inc\RTSPServerControl.h"
+using namespace winrt;
+using namespace Windows;
+using namespace Media::Capture;
+using namespace Frames;
+using namespace Windows::Graphics::Imaging;
+using namespace Windows::Foundation;
+using namespace Windows::Devices::Enumeration;
+using namespace Windows::Media;
+using namespace Windows::Media::Devices;
+using namespace Windows::Media::MediaProperties;
+constexpr uint16_t ServerPort = 8554;
+constexpr uint16_t SecureServerPort = 6554;
+
+// Uncomment the following if you want to use FrameReader API instead of the Record-to-sink APIs
+//#define USE_FR 
+

```

Comment:
Could this be cmd line parameter instead of something that requires recompile ?

Reply:
i thought it was easy to understand code if one collapses all the FR stuff.  Since this is just a test application, any particular reason why this might be useful as runtime param?

---
In reply to: [491090080](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491090080) [](ancestors = 491090080)

Reply:
No particular reason, just a thought. I'm fine like this :)

---
In reply to: [491220249](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491220249) [](ancestors = 491220249,491090080)